pytorch_lightning/plugins/precision/native_amp.py

```diff
@@ -88,6 +89,13 @@ def post_optimizer_step(self, optimizer: Optimizer, optimizer_idx: int) -> None:
         self.scaler.step(optimizer)
         self.scaler.update()
 
+    def move_grad_to_cpu(self, model):
+        if hasattr(model, "cpu_offload"):
+            if model.cpu_offload:
+                for param in model.params:
+                    param._cpu_grad.copy_(param.grad.data, non_blocking=True)
+                    param.grad.data = param._cpu_grad
```

Comment:
```suggestion
        if getattr(model, "cpu_offload", False):
            for param in model.params:
                param._cpu_grad.copy_(param.grad.data, non_blocking=True)
                param.grad.data = param._cpu_grad
```

Reply:
@justusschock Just curious, what's wrong with `hasattr(..)`?

Reply:
Nothing wrong, just not mypy compatible :) and also we can save a check here :)